packages/gui/src/components/vcs/VCCard.tsx

```diff
@@ -44,7 +43,7 @@ export default function VCCard(props: { vcRecord: any; isDetail?: boolean; proof
   const [revokeVC] = useRevokeVCMutation();
   const theme: any = useTheme();
   const openDialog = useOpenDialog();
-  const [vcTitlesObject] = usePrefs<any>('verifiable-credentials-titles', {});
+  const [vcTitlesObject] = useLocalStorage<any>('verifiable-credentials-titles', {});
```

Comment:
Why are we switching from prefs to local storage? Won't we lose existing data from prefs if we make this change?

Reply:
A huge mistake on my end. I thought if we're storing VCs in localStorage, then titles shouldn't be in prefs.
However I completely forgot about onchain VCs :/